docs/content/latest/releases/whats-new/stable-releases.md

```diff
@@ -21,6 +20,85 @@ Included here are the release notes for all releases in the v2.4 stable release
+* [BACKPORT 2.4] [[6960](https://github.com/yugabyte/yugabyte-db/issues/6960)] docdb - Register ScopedRWOperation when accessing doc_db data in StillHasParentDataAfterSplit
+* [BACKPORT 2.4] [[7105](https://github.com/yugabyte/yugabyte-db/issues/7105)] Explicitly clear the global PgMemctx map in YBCDestroyPgGate
+* [BACKPORT 2.4] [[7145](https://github.com/yugabyte/yugabyte-db/issues/7145)] Fix wrong propagated safe time
+* [BACKPORT 2.4] [[7289](https://github.com/yugabyte/yugabyte-db/issues/7289)] Test split in progress should not block LB data move if blacklist is set
+* [BACKPORT 2.4] [[7355](https://github.com/yugabyte/yugabyte-db/issues/7355)] ysql: check backfill bad connection status
+* [BACKPORT 2.4] [[7397](https://github.com/yugabyte/yugabyte-db/issues/7397)] Tablet splitting: Crash during generating TSHeartbeat report after split
+* [BACKPORT 2.4] [[7705](https://github.com/yugabyte/yugabyte-db/issues/7705)] ysql: prioritize internal HBA config
+* [BACKPORT 2.4] [[7835](https://github.com/yugabyte/yugabyte-db/issues/7835)] Don't crash when trying to append ValueType::kTombstone to a key
+* [Backport 2.4] Update NFS shared path /n to /Volumes/n
+* [Backport 2.4] [[5647](https://github.com/yugabyte/yugabyte-db/issues/5647)] docdb: LB should not move tablets right after splitting
+* [Backport 2.4] [[7054](https://github.com/yugabyte/yugabyte-db/issues/7054)] Fix undefined variable in ListTables conditional check.
+* [Backport 2.4] [[7079](https://github.com/yugabyte/yugabyte-db/issues/7079)] [[7153](https://github.com/yugabyte/yugabyte-db/issues/7153)] YSQL: Improve configuration and defaults for client to server encryption
+* [Backport 2.4] [[7455](https://github.com/yugabyte/yugabyte-db/issues/7455)] [YCQL] Update index from transaction with cross-key statements.
+* [Backport 2.4] [[7628](https://github.com/yugabyte/yugabyte-db/issues/7628)] Add ldap libraries as special case for yb client packaging
+* [Backport 2.4] [[7641](https://github.com/yugabyte/yugabyte-db/issues/7641)] YCQL: Fix checks in index update path that determine full row removal.
+* [Backport 2.4] [[7769](https://github.com/yugabyte/yugabyte-db/issues/7769)] Prevent adding on-prem node instance with duplicate IP
+* [Backport 2.4] [[7813](https://github.com/yugabyte/yugabyte-db/issues/7813)] [YSQL] YSQL dump should always include HASH/ASC/DESC modifier for indexes/pkey.
+
+#### Yugabyte Platform
+
+* [[7416](https://github.com/yugabyte/yugabyte-db/issues/7416)] [Platform] Changed default port of On-Prem provider to 22 (#7599)
+* [[7704](https://github.com/yugabyte/yugabyte-db/issues/7704)] [Platform] Backup to S3 fails using Yugaware instance's IAM role
+* [BACKPORT 2.4] [[6668](https://github.com/yugabyte/yugabyte-db/issues/6668)] Update platform backup script to latest revision
```

Comment:
Updating the script doesn't quite describe the feature (though it was part of it). Can we change this to read:
```
[BACKPORT 2.4] [[6668](https://github.com/yugabyte/yugabyte-db/issues/6668)] Support platform backup/restore on Kubernetes
```